pkg/handlers/generic/lifecycle/csi/handler.go

```diff
@@ -106,84 +98,42 @@ func (c *CSIHandler) AfterControlPlaneInitialized(
 		)
 		return
 	}
+	if len(csiProviders.Providers) == 1 &&
```

Comment:
What do we want to do when the user configures two or more CSI providers, but does not configure a default storage? We can't choose one. Do we omit a default storage? Do we return an error?

Reply:
We should omit a default storage class i guess. it doesn't seem worthy of throwing an error